resources/views/layouts/app.blade.php

```diff
@@ -15,27 +15,32 @@
                         </a>
                     </div>
                     <div class="flex-1 text-right">
                         @guest
-                            <a class="no-underline hover:underline text-gray-300 text-sm p-3" href="{{ route('login') }}">{{ __('Login') }}</a>
+                            <a class="no-underline hover:underline text-black text-sm p-3" href="{{ route('login') }}">{{ __('Login') }}</a>
                             @if (Route::has('register'))
-                                <a class="no-underline hover:underline text-gray-300 text-sm p-3" href="{{ route('register') }}">{{ __('Register') }}</a>
+                                <a class="no-underline hover:underline text-black text-sm p-3" href="{{ route('register') }}">{{ __('Register') }}</a>
                             @endif
                         @else
-                            <span class="text-gray-300 text-sm pr-4">{{ Auth::user()->name }}</span>
+                            <a class="inline-block py-2 px-4 text-black font-bold no-underline" href="/home">Dashboard</a>
+
+                            <a class="inline-block text-black no-underline hover:text-gray-800 hover:text-underline py-2 px-4" href="/quiz">Quiz</a>
+
+                            
+                            <img class="inline-block rounded-full h-15 w-15 mr-4" src="https://cdn.i-scmp.com/sites/default/files/styles/768x768/public/d8/images/methode/2019/11/04/37716de6-fee6-11e9-93ee-a5388fc1b87d_image_hires_174946.jpg?itok=81dBdlex&v=1572860994">
```

Comment:
What's the cdn referring to here? Is this an icon? If so we would use an svg say from heroicons.